docs/delete-product-or-product-variant-metafields-in-bulk/script.liquid

```diff
@@ -148,7 +101,7 @@
 
     {% assign bulkOperation = hash %}
     {% assign bulkOperation["objects"] = objects_jsonl | parse_jsonl %}
-
+    
```

Comment:
Trailing whitespace! Validation for this really belongs in this project's tests, but - please remove these spare spaces, and then we're done! :D

Reply:
Done :)